ui/src/app/components/buttons.tsx

```diff
@@ -117,19 +117,17 @@ const computeStyle = ({style = {}, hover = {}, disabledStyle = {}}, {disabled})
 
 // Set data test id = '' in the child to prevent it from propegating to all children
 export const Clickable = ({as = 'div', disabled = false, onClick = null, ...props}) => {
-  // Don't propagate test IDs to the rendered child component.
-  delete props['data-test-id'];
   return <Interactive
-    as={as} {...props}
+    // `fp.omit` used to prevent propagation of test IDs to the rendered child component.
+    as={as} {...fp.omit(['data-test-id'], props)}
```

Comment:
I want to make a general function like `removeDataTestId(props)` to make this a bit more clear.

Reply:
That would be reasonable – but actually the fp.omit is pretty clear & concise as it stands.